src/vs/workbench/contrib/terminal/browser/terminalActions.ts

```diff
@@ -1711,6 +1711,26 @@ export function registerTerminalActions() {
 			}
 		}
 	});
+	registerAction2(class extends Action2 {
+		constructor() {
+			super({
+				id: TerminalCommandId.ShowOrFocusHover,
+				title: terminalStrings.showOrFocusHover,
+				f1: false,
```

Comment:
Don't we want this to be in the command palette for discoverability? The editor one is